docs/typedoc/intro.md

```diff
@@ -0,0 +1,11 @@
+# Ably Spaces SDK
+
+<!-- TODO needs proper content -->
+
+This is the documentation for Ably’s Spaces SDK. Start with the documentation for the SDK’s {@link default | default export}.
+
+> **Note**
+>
+> This documentation is a work in progress and will be improved over time.
+
+For alternative documentation, see our [documentation website](https://ably.com/docs/spaces).
```

Comment:
Is this file necessary (i.e. is included in the generated output on the remote)?

Reply:
Yeah, take a look at the generated HTML documentation and you'll see that it's the landing page 

(See the `readme` configuration in `typedoc.json`. If you don't specify one, then it'll use the repo’s `README`, which I thought looked a bit odd. We do the same thing — custom start page for generated docs — [in ably-js](https://sdk.ably.com/builds/ably/ably-js/main/typedoc/default/index.html), too.)

Reply:
I see, thanks. Can we please change the contents to the following:

```
# Ably Spaces SDK

This is the API reference for Ably’s Spaces SDK, generated with [typedoc](https://typedoc.org/). 

To find more about Spaces and how to get started head to [Ably documentation website](https://ably.com/docs/spaces).
```

Reply:
OK, have done (with some tweaks to the wording).